hack/make/.build-empty-images

```diff
@@ -24,3 +29,27 @@ if ! docker image inspect emptyfs > /dev/null; then
 	)
 	rm -rf "$dir"
 fi
+
+# without c8d image store, image id is the config's id
+dangling_cfg=7288130afe9d3bcac1dc67b1e1f899e11ddaddfd40f6cd9da0fadeccb35a72d0
+# with c8d image store, image id is the id of manifest/manifest list.
+dangling_mfst=019609c4281fb13dc186fb1e636a38cb4e2e57b156bdbf68d2944314119d3d15
+if imageNotPresent "$dangling_cfg" && imageNotPresent "$dangling_mfst"; then
+	dir="$DEST/dangling"
+	mkdir "$dir"
+	(
+		cd "$dir"
+		echo -n '{"schemaVersion":2,"manifests":[{"mediaType":"application/vnd.docker.distribution.manifest.v2+json","digest":"sha256:019609c4281fb13dc186fb1e636a38cb4e2e57b156bdbf68d2944314119d3d15","size":264,"annotations":{"org.opencontainers.image.created":"2023-05-19T08:00:44Z"},"platform":{"architecture":"amd64","os":"linux"}}]}' > index.json
+		echo -n '[{"Config":"blobs/sha256/7288130afe9d3bcac1dc67b1e1f899e11ddaddfd40f6cd9da0fadeccb35a72d0","RepoTags":null,"Layers":null}]' > manifest.json
+		mkdir -p blobs/sha256
+		echo -n '{"schemaVersion":2,"mediaType":"application/vnd.docker.distribution.manifest.v2+json","config":{"mediaType":"application/vnd.docker.container.image.v1+json","digest":"sha256:7288130afe9d3bcac1dc67b1e1f899e11ddaddfd40f6cd9da0fadeccb35a72d0","size":340},"layers":[]}' > blobs/sha256/$dangling_mfst
+		echo -n '{"architecture":"amd64","config":{"Env":["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"],"WorkingDir":"/","Labels":{"dangling":"1"},"OnBuild":null},"created":null,"history":[{"created_by":"LABEL dangling=1","comment":"buildkit.dockerfile.v0","empty_layer":true}],"os":"linux","rootfs":{"type":"layers","diff_ids":null}}' > blobs/sha256/$dangling_cfg
```

Comment:
could it be related to platform? although I think "empty" images would default to `linux/amd64`

Reply:
This image is `linux/amd64` explicitly. It also passed on my `amd64` machine, but I only run that one particular test.
IMO, there's some other integration test which does prune or other operation, and the image is gone by the time the new test gets the chance to run.

Reply:
Done